.yamllint_defaults.yml

```diff
@@ -13,4 +13,6 @@ rules:
   truthy:
     allowed-values: ["yes", "no", "true", "false"]
     level: error
+  line-length:
+    level: warning
```

Comment:
What is the reason for this change?  We want line-length to be an error for regular roles, and we want to override it to be ignored for collections

Reply:
Yes, true... I'm going to revert this change and rethink about it.